All_years/04-Social_Care/04-SDS_functions.R

```diff
@@ -0,0 +1,70 @@
+#' Open a connection to a PHS database
```

Comment:
I think we should commit these functions to the sep update branch into a new script called `Social_Care_functions.R` as there will be a lot of overlap between these, the existing Home Care ones and the A&T ones you have on the other branch.

That will also make it easier to convert as all the functions will be in one place (and there won't be any duplicates of them).